cmd/astrolabe/templates/base.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="referrer" content="origin-when-cross-origin">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="color-scheme" content="light dark" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@picocss/pico@2/css/pico.blue.min.css" />
+	<style>
+		html { position: relative; min-height: 100%; height: auto; }
+		body { margin-bottom: 3em; }
+		body > nav { background-color: var(--pico-muted-border-color); }
+		body > footer { position: absolute; bottom: 0px; padding: 2em; background-color: var(--pico-muted-border-color); }
+		thead th { font-weight: bold; }
+		main article { margin: 2.5rem 0; padding: 2rem; }
+	</style>
+	<meta name="generator" name="astrolabe">
+  <title>{% block head_title %}astrolabe{% endblock %}</title>
+</head>
+<body>
+<nav class="container-fluid">
+  <ul>
+  	<li><a href="/"><strong>astrolabe</strong></a></li>
+  </ul>
+	<form action="/query" method="get" style="width: 80%;">
+  	<input type="text" name="q" placeholder="at://..." style="margin: 0.5em;">
+	</form>
+  <ul>
+    <li><a href="https://github.com/bluesky-social/indigo">Code</a></li>
+  </ul>
+</nav>
+
+<main class="container">
+	{% block main_content %}Base Template{% endblock %}
+	<br>
+	<br>
+	<br>
+</main>
+
+<footer class="container-fluid">
+	<div class="container">
+    <small>Developed by <a href="https://blueskyweb.xyz">Bluesky PBC</a> for <a href="https://atproto.com">atproto</a></small>
```

Comment:
```suggestion
    <small>Developed by <a href="https://bsky.social">Bluesky PBC</a> for <a href="https://atproto.com">atproto</a></small>
```
nit: suggest updating Bluesky PBC website to `bsky.social`

Reply:
thanks! that is a better URL, I should update the did-method-plc homepage which I copied this from.

for astrolabe, I might just delete this entire footer entirely